migration/wcmp-upgrade-migration-v4-2-1.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+use migration\WCMP_Upgrade_Migration;
+
+if (! defined('ABSPATH')) {
+    exit;
+}
+
+if (class_exists('WCMP_Upgrade_Migration_v4_2_1')) {
+    return new WCMP_Upgrade_Migration_v4_2_1();
```

Comment:
Moet je hier niet alleen `return;` doen? Je gebruikt hem namelijk niet in:
![image](https://user-images.githubusercontent.com/16726304/108066578-cf79c480-705f-11eb-8027-60210ff87d89.png)

Reply:
Eigenlijk zou je het zelfde moeten doen als woocommerce en dat is de `require_once` binnen de if met de `class_exists ` moeten zetten.

maar om alleen de de return neer te zetten zoals je zei, dat gaat niet werken